Fix description of ASPNETCORE_HTTPS_PORT (#33549)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/host/generic-host.md

### HTTPS_Port

> The HTTPS redirect port. Used in [enforcing HTTPS](xref:security/enforcing-ssl). This setting doesn't also cause the server to listen on that port. That is, it's possible to accidentally redirect requests to an unused port.

Reviewer: That's a little bit clumsy, something like

```suggestion
The HTTPS redirect port. Used in [enforcing HTTPS](xref:security/enforcing-ssl). Like xyz, with this setting,  the server doesn't listen on the specified port. That is, it's possible to accidentally redirect requests to an unused port.
```

Author: How about this:

```suggestion
Set the HTTPS port to redirect to if you get a non-HTTPS connection. Used in [enforcing HTTPS](xref:security/enforcing-ssl). This setting doesn't cause the server to listen on the specified port. That is, it's possible to accidentally redirect requests to an unused port.
```
